### DISCUSSION TITLE
Path stripping from proxy?

### DISCUSSION THREAD

**Discussion by Hajitorus** [digitallyinduced/ihp#1948] [General]
When I'm running up dev services, I often run them on localhost and put them behind a proxy for access to hosts on my local network. The easy way for me to do this is differentiating by path; for example, I tried to set up ihp and the admin interface like this:

`https://myproxy/ihp/` -> `localhost:8000`
`https://myproxy/ihp-admin/` -> `localhost:8001`

But this means that resources aren't loaded correctly, since IHP sends HTML payloads that reference e.g. `/ihp-welcome-icon.svg` instead of `/ihp/ihp-welcome-icon.svg`. I tried setting `IHP_BASEURL`, but I can see from the source code that any paths are explicitly stripped off before being handed to the rest of the code. When I've set up services like Gitea in the past, setting the `BASEURL` (or analog) allowed this proxying scheme to work.

For now I'm just giving over the entire proxy host to ihp while I'm playing, so this isn't a giant issue. Am I missing a giant sequined document somewhere that points out exactly what I'm doing wrong? I've tried searching the docs a few different ways but wasn't able to find anything.

**Comment by mpscholten** [comment]
`IHP_BASEURL` is supposed to solve this. We should remove the path stripping IMO
